code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -114,6 +114,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	..()
 	flags |= NOREACT // so it doesn't react until you light it
 	create_reagents(chem_volume) // making the cigarette a chemical holder with a maximum volume of 15
+	reagents.add_reagent("nicotine", 15)

```

Comment:
Sorry, but wouldn't this mean you can no longer dunk cigarettes in other chemicals since they start off full?